front/tests/utils/api-setup.ts

```diff
@@ -260,3 +264,46 @@ export async function setStdcmEnvironment(stdcmEnvironment: StdcmSearchEnvironme
     'Failed to update STDCM configuration environment'
   );
 }
+
+/**
+ * Retrieve a towed rolling stock by name.
+ *
+ * @param towedRollingStockName - The name of the towed rolling stock to retrieve.
+ * @returns {Promise<TowedRollingStock >} - The matching towed rolling stock data .
+ */
+export const getTowedRollingStock = async (
+  towedRollingStockName: string
+): Promise<TowedRollingStock | undefined> => {
+  const towedRollingStocks: GetTowedRollingStockApiResponse = await getApiRequest(
+    '/api/towed_rolling_stock',
+    { page_size: 50 }
+  );
+  const towedRollingStock = towedRollingStocks.results.find(
+    (t: TowedRollingStock) => t.name === towedRollingStockName
+  );
+  return towedRollingStock as TowedRollingStock;
+};
+
+/**
+ * Creates a towed rolling stock using predefined data from the imported JSON file.
+ *
+ * @returns {Promise<TowedRollingStock>} - The created towed rolling stock.
+ */
+export async function setTowedRollingStock(): Promise<TowedRollingStock> {
```

Comment:
It's better to check by name because, if there are other tower rolling stocks besides the one designated for the E2E test, your logic might fail. This is especially important when running the tests locally with the dump